### PR TITLE
helm chart fixes

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- add double underscore to helm chart settings
- 
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by pbolduc** [comment]
👀

File: .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml [hunk 1]
```diff
@@ -50,17 +50,17 @@ spec:
...
           - name: ASPNETCORE_URLS
             value: http://*:8080;http://*:9090
-          - name: COMS_USERNAME
+          - name: COMS__USERNAME
             valueFrom:
               secretKeyRef:
                 name: {{ .Values.coms.service.secretName }}
                 key: username
-          - name: COMS_USERNAME
+          - name: COMS__USERNAME
```
Comment (pbolduc) [inline .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml:58]: duplicate entry

**Review by pbolduc — CHANGES_REQUESTED** [review]
just remove the duplicate username

**Review by KDNeufeld — APPROVED** [review]
Yes, correct. .NET separates keywords by double underscore. (Java uses the typical dot notation separator).

**Pull request merged**